src/Symfony/Bundle/FrameworkBundle/Resources/config/property_access.xml

```diff
@@ -4,10 +4,22 @@
     xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
     xsi:schemaLocation="http://symfony.com/schema/dic/services http://symfony.com/schema/dic/services/services-1.0.xsd">
 
+    <parameters>
+        <parameter key="property_accessor.cache.prefix" />
+    </parameters>
+
     <services>
         <service id="property_accessor" class="Symfony\Component\PropertyAccess\PropertyAccessor" >
             <argument /> <!-- magicCall, set by the extension -->
             <argument /> <!-- throwExceptionOnInvalidIndex, set by the extension -->
+            <argument /> <!-- cache, set by the extension -->
+        </service>
+
+        <!-- Cache -->
+        <service id="property_accessor.cache.apc" class="Doctrine\Common\Cache\ApcCache" public="false">
```

Comment:
Ok got it. One argument to keep filesystem adapters in FrameworkBundle instead of the SE: it becomes possible to enable them everywhere by default and it will work for apps using the new Microkernel or other editions too. There are a lot of people that never remove comments about APC cache in config_prod.yml just because they are not aware of this feature. Having optims enabled by default in all cases (and even with the microkernel) can be nice if we rely on something always available like the FS cache.

What do you think?